WooCommercePOS/WooCommercePOS/Presentation/CartView.swift

```diff
@@ -9,14 +9,21 @@ struct CartView: View {
 
     var body: some View {
         VStack {
-            ProductRowView()
-            ProductRowView()
-            ProductRowView()
+            Text("Cart")
+                .foregroundColor(Color.white)
```

Comment:
We will need another semantic colour here, since `Color.primaryText` in light mode will currently default to `.black`

Reply:
Totally fine to leave the colors/styles for later when we have a more concrete design 👍